mock-mhs-adaptor/src/main/java/uk/nhs/adaptors/mockmhsservice/service/MockMhsService.java

```diff
@@ -77,8 +78,15 @@
         } else if (interactionId.equals(ACKNOWLEDGEMENT_INTERACTION_ID)) {
             LOGGER.info("Message acknowledgement accepted.");
             headers.setContentType(MediaType.TEXT_XML);
+            try{
+                var inboundMessage = STUB_ACKNOWLEDGEMENT_INBOUND_MESSAGE.replace("%%ConversationId%%", correlationId);
+                inboundProducer.sendToMhsInboundQueue(inboundMessage);
+                LOGGER.info("Message acknowledgement sent to Inbound Queue, acknowledgementInteractionId: " + ACKNOWLEDGEMENT_INTERACTION_ID);
+            }catch (JmsException e) {
+                LOGGER.error("Error could not send acknowledgement to Inbound Queue", e);
```

Comment:
I think we should return an error when exception is caught. At the moment you're only logging it and then returning HTTP 202

Reply:
Still need to return a 202 here